_docs-sources/pipelines/upgrading/upgrading-from-config-yml.md

```diff
@@ -0,0 +1,106 @@
+# Upgrading from config.yml
+
+Previously, certain configurations for Pipelines were defined within a `.gruntwork/config.yml` file located at the root of repositories using Pipelines.
+
+To improve the quality of the Pipelines experience, configurations are now defined in the Pipelines Configuration as Code system. This system allows for more flexibility and control over configurations, allowing Pipelines to be used in a wider variety of use-cases, for a larger number of users.
+
+:::note
+YAML configurations are read by Pipelines when HCL configurations are not present, and the Pipelines binary falls back to interpretting YAML configurations as if they were defined in the HCL configuration system in this scenario.
+
+This means that if you have a `.gruntwork/config.yml` file in your repository, you can continue to use it as you have been, and Pipelines will continue to work as expected. If you do introduce any HCL configurations into your `.gruntwork` directory or introduce a `gruntwork.hcl` file into a directory, Pipelines will begin to use the HCL configuration system instead of the YAML configuration system.
+:::
+
+## Should I Migrate?
+
+At this time, there is no need to migrate from YAML configurations to HCL configurations. Pipelines will continue to work as expected with YAML configurations.
+
+Gruntwork does not currently recommend migrating from YAML configurations to HCL configurations. There may be a future point where YAML configurations are deprecated, but there will be robust tooling and documentation to support this migration when it becomes necessary.
+
+## YAML Configurations
```

Comment:
It's a bit misleading to me that the docs for the YAML config live in a page called "Upgrading from Yml" -- IMO it should still live as its own first class page during this transition period